helpers.sh

```diff
@@ -76,16 +76,22 @@ function repo_sync {
 	fi
 	_repo_extra_args=""
 	for i in $(seq 4); do
-		run repo init $_repo_extra_args --repo-rev=v2.35 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES} && break
+		if run repo init $_repo_extra_args --repo-rev=v2.35 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES}; then
```

Comment:
There is another way to always return true in such commands:
`		run repo init $_repo_extra_args --repo-rev=v2.35 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES} && break || true`

What do you think about this way?

Reply:
using `if` for this looks better to me and is more clean